src/PathTranslator.ts

```diff
@@ -193,7 +209,7 @@ export class PathTranslator {
 				pathInfo.fileName = INIT_NAME;
 			}
 
-			pathInfo.exts.push(LUA_EXT); // push .lua
+			pathInfo.exts.push(this.getLuauExt()); // push .luau
```

Comment:
This is not backwards compatible. Packages that still use `.lua` would not be resolved correctly.

Reply:
A realization I had while writing this:

RojoResolver stores references to `.lua` file paths exactly if a `$path` points to a `.lua` file. Will need to find a solution here.